components/engine_panic/src/raft_engine.rs

```diff
@@ -144,6 +144,10 @@ impl RaftEngine for PanicEngine {
         panic!()
     }
 
+    fn get_engine_path(&self) -> &str {
```

Comment:
```suggestion
    fn path(&self) -> &str {
```

Reply:
KV engine also have an interface `path` in misc trait - that may introduce some ambiguous name problems

Reply:
TODO is added.